tests/runner.py

```diff
@@ -1137,7 +1137,6 @@ def do_run(self, src, expected_output, args=[], output_nicerizer=None,
              check_for_error=True):
     if force_c or (main_file is not None and main_file[-2:]) == '.c':
       basename = 'src.c'
-      Building.COMPILER = to_cc(Building.COMPILER)
```

Comment:
This was causing a non-local changes which previously prevented the bullet test from failing when run in combination with other tests.

This explains why this went unnoticed: https://github.com/emscripten-core/emscripten/pull/10951!